ci/lint/lint.sh

```diff
@@ -10,6 +10,10 @@ clang_format() {
   ./ci/lint/check-git-clang-format-output.sh
 }
 
+clang_tidy() {
+  ./ci/lint/check-git-clang-tidy-output.sh
```

Comment:
wow just curious, can we directly use clang-tidy by running this script?
Asking this because I tried to do that on my devbox, and I remember I had to generate a compilation database beforehand.

Reply:
Unfortunately, it won't work. I'm just experimenting. Have you ever successfully run this script?

Reply:
Ah gotcha; no I didn't know the script, but a month ago I successfully ran clang tidy on devbox, and it requires some setups